docs/ide/README.md

```diff
@@ -4,7 +4,7 @@ Your IDE is your primary tool while working with code, we recommend the followin
 
 ## Visual Studio Code
 
-We recommend [VSCode](https://code.visualstudio.com/).
+We recommend [VSCode](https://code.visualstudio.com/). For alternative IDEs see  [here](alternatives.md)
```

Comment:
```suggestion
We recommend [VSCode](https://code.visualstudio.com/). For alternative IDEs see [here](alternatives.md).

Reply:
Very nitpicky but can we add a period at the end of the sentence?